CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
       // New
       err = r.client.List(context.TODO(), podList, client.UseListOptions(listOps))
       ```
+- CRD file names were previously of the form `<group>_<version>_<kind>_crd.yaml`. Now that CRD manifest `spec.version` is deprecated in favor of `spec.versions`, i.e. multiple versions can be specified in one CRD, CRD file names have the form `<full group>_<resource>_crd.yaml`. `<resource>` is the plural lower-case CRD Kind found at `spec.names.plural`.
```

Comment:
Can we point out the difference between group vs full group here e.g: `foo.bar.com` is the full group and `foo` is the shortened group.
Remember only we make that distinction in the SDK.
The [k8s API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md) refer to it as just group.
```
Group names are typically in domain name form.
When choosing a group name, we recommend selecting a subdomain your group or organization owns, such as "widget.mycompany.com".
Group names must be lower case and be valid DNS subdomains
```